amt/site/templates/projects/details_info.html.j2

```diff
@@ -23,20 +23,18 @@
             <tr class="rvo-table-row">
                 <td class="rvo-table-cell">{% trans %}Lifecyle{% endtrans %}</td>
                 <td class="rvo-table-cell">
-
-                    <div
-                        target="_self"
-                        class="rvo-tag rvo-tag--with-icon rvo-tag--active rvo-tag--success"
-                    >
-  <span
-      class="utrecht-icon rvo-icon rvo-icon-bevestiging rvo-icon--lg"
-      role="img"
-      aria-label="Bevestiging"
-  ></span>
-                        {{ lifecycle.name }}
+                    <div target="_self" class="rvo-tag rvo-tag--with-icon rvo-tag--active rvo-tag--success">
+                      <span class="utrecht-icon rvo-icon rvo-icon-bevestiging rvo-icon--lg" role="img"></span>
+                      {{ lifecycle.name }}
                     </div>
                 </td>
             </tr>
+            <tr class="rvo-table-row">
+                <td class="rvo-table-cell">{% trans %}Last updated{% endtrans %}</td>
+                <td class="rvo-table-cell">
+                    {{ last_edited|format_datetime("en") }}
```

Comment:
@anneschuth can we make the formatting configurable based on the user's language setting?

Reply:
Great question, I tried but failed initially. Let me try again.

Reply:
@anneschuth did you try:


```suggestion
                    {{ last_edited|format_datetime(language) }}
```

Or maybe we can use the full version?

```suggestion
                    {{ last_edited|format_datetime(language, "full") }}
```

Reply:
yes, and that works :)  It's already in this branch.